frontend/src/app/modules/home/projects/edit/project-general/project-general.component.html

```diff
@@ -0,0 +1,61 @@
+<p-panel header="General" class="panel">
+    <form [formGroup]="editForm" class="item">
+        <label class="item-definition">
+            <div>Name</div>
+            <div class="item-sub-text">Name of your project</div>
+        </label>
+        <div class="item-field">
+            <div class="item-form-input p-input-icon-right">
+                <i *ngIf="name?.pending" class="pi pi-spin pi-spinner"></i>
+                <input formControlName="name" pInputText placeholder="Project name" class="item-form-field">
```

Comment:
It's not necessary, but you can use this `[maxLength]="50"`, to limit user input. If you write this, user can't write more than 50 symbols in this input.

Reply:
Ok i did it

Reply:
I'm sorry, forget about this. Just because 90% of our inputs not use this attribute